### PR TITLE
Adjust the partitioning by outcome.

### PR DESCRIPTION
**Pull request by beorn7**
@juliusv

### CODE REVIEW

File: notification/notification.go [hunk 1]
```diff
@@ -103,15 +100,24 @@ func NewNotificationHandler(alertmanagerURL string, notificationQueueCapacity in
...
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "errors_total",
+			Help:      "Total number of errors sending alert notifications.",
+		}),
+		notificationDropped: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "dropped_total",
+			Help:      "Total number of dropped alert notifications.",
```
Comment (brian-brazil) [inline notification/notification.go]: Mention that this is due to the alertmanager not being configured maybe?

**Comment by brian-brazil** [comment]
:+1:

File: notification/notification.go (hunk 1 above)
Comment (beorn7) [inline notification/notification.go]: Good idea.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**